src/components/designer/taro/actions/NewAddressButton.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { QrcodeOutlined } from '@ant-design/icons';
+import styled from '@emotion/styled';
+import { Button, Form } from 'antd';
+import { usePrefixedTranslation } from 'hooks';
+import { TaroNode } from 'shared/types';
+import { useStoreActions } from 'store';
+
+const Styled = {
+  Button: styled(Button)`
+    width: 100%;
+  `,
+};
+interface Props {
+  node: TaroNode;
+  type?: 'button' | 'menu';
+}
+
+const OpenSendAssetModal: React.FC<Props> = ({ node, type }) => {
+  const { l } = usePrefixedTranslation('cmps.designer.taro.actions.NewAddressButton');
+  const { showNewAddress } = useStoreActions(s => s.modals);
+  const btnlabel = l('newAddress');
```

Comment:
This is super nitpicky but I can be pretty anal about consistency in a code base. :)

I'd prefer to keep the syntax identical to the other Button components. There isn't much benefit to declaring this var here versus placing the value in the JSX below.

Reply:
done

Reply:
done.